plugin-unmanaged-racks/README.md

```diff
@@ -0,0 +1,164 @@
+<!-- 
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+# Unmanaged Racks Plugin (URP) 
+
+This folder contains implementation of Unamanaged Racks Plugin(URP) for ODIMRA. 
+Plugin exposes narrowed obligatory REST API described by Plugin Developer’s Guide (PDG).
+In addition URP exposes following REST endpoints:
+
+* `GET /ODIM/v1/Chassis` - return collection of unmanaged Chassis(RackGroups/Racks)
+* `GET /ODIM/v1/Chassis/{id}` - return instance of unmanaged Chassis(RackGroups/Racks)
+* `POST /ODIM/v1/Chassis` - creates new unmanaged Chassis(RackGroups/Racks) 
+* `DELETE /ODIM/v1/Chassis/{id}` - deletes existing unmanaged Chassis(RackGroups/Racks)
+* `PATCH /ODIM/v1/Chassis/{id}` - updates existing unmanaged Chassis(RackGroups/Racks)
+
+Full specification of URP is available here: https://wiki.odim.io/display/HOME/Plugin+for+Unmanaged+Racks.
+
+Please be aware this plugin is still under development, and some features might be missing.
+
+## Build 
+
+Build URP using following command:
+```
+cd plugin-unmanaged-racks
+make build
+``` 
+
+Run URP using run target:
+```
+make run
+```
+
+## Register URP in ODIMRA
+
+1. Make `https://localhost:45000/redfish/v1/AggregationService/ConnectionMethods` endpoint exposes connection method required by URP plugin. `ConnectionMethodVariant` should be `Compute:BasicAuth:URP_v1.0.0`.
```

Comment:
IMHO instead of mentioning `https://localhost:45000` or `https://odimra.local.com:45000` same can be mentioned like `https://{odim_host}:{port} `

Reply:
Done.